tests/components/mqtt/test_init.py

```diff
@@ -1197,7 +1197,6 @@ async def test_mqtt_ws_get_device_debug_info(
     }
     data_sensor = json.dumps(config_sensor)
     data_trigger = json.dumps(config_trigger)
-    config_sensor["platform"] = config_trigger["platform"] = mqtt.DOMAIN
```

Comment:
This key is no longer added, see:
https://github.com/home-assistant/core/pull/118757/files#diff-189ecd07b98c2a9602a647f97dfe63e919c95af2d3856701044958b230591ef1L274